Description: Enable mipsel build
 wscript needs to recognize mips arch.
 tools/test.py is only used to run js tests, so arch is not needed there.
Forwarded: not-needed
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2015-01-28
--- a/tools/test.py
+++ b/tools/test.py
@@ -1203,12 +1203,12 @@
         return False
     # Ensure that the simulator argument is handed down to scons.
     options.scons_flags.append("simulator=" + options.simulator)
-  else:
+  #else:
     # If options.arch is not set by the command line and no simulator setting
     # was found, set the arch to the guess.
-    if options.arch == 'none':
-      options.arch = ARCH_GUESS
-    options.scons_flags.append("arch=" + options.arch)
+    # if options.arch == 'none':
+    #  options.arch = ARCH_GUESS
+    # options.scons_flags.append("arch=" + options.arch)
   if options.snapshot:
     options.scons_flags.append("snapshot=on")
   return True
